packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,264 @@
+# Terra Application Layout
+
+## Usage Summary
+
+1. The ApplicationLayout requires a `Base` component (from `terra-base`) and any `Router` component (from `react-router-dom`) in its parent hierarchy.
+2. The `nameConfig` prop customizes the ApplicationLayout's branding.
+3. The `utilityConfig` prop generates an utilities menu and presentation controls within the layout. A default set of options can be generated by a helper.
+4. The `routingConfig` prop determines which components get rendered in the ApplicationLayout based on the current router location. Usage of the path `'/'` is restricted for `menu` components.
+5. The `navigationItems` prop generates primary navigation controls within the layout. The provided navigation items should have associated content in the `routingConfig`.
+6. The `indexPath` prop determines the default redirect location for the layout. The provided path should have associated content in the `routingConfig`.
+7. Components rendered within the ApplicationLayout also receive an `app` prop with default support for modal disclosures.
+8. Components rendered within the ApplicationLayout receive a `layoutConfig` prop for layout state manipulation.
+9. Components rendered within the ApplicationLayout also receive an `routingStackDelegate` prop.
+10. The `show` and `showParent` functions of the `routingStackDelegate` should be used for virtual navigation within configuration regions. Specifically, `showParent` must be implemented by `menu` components to ensure that access to the `ApplicationLayout`'s default menu is maintained.
+
+## Prerequisites
+
+- Like all Terra components, the ApplicationLayout requires the presence of a `Base` component (provided by `terra-base`) in its parent hierarchy.
```

Comment:
You listed this in bullet number 1 as well. Also should we list these before the usage?

Reply:
The summary section is meant to summarize the rest of the page. I can move it to the bottom if leading with it is confusing, or I can remove it altogether if it's not adding value.